apps/preview/next/pages/showcases/MegaMenu/BaseMegaMenu.tsx

```diff
@@ -219,9 +235,9 @@ export default function BaseMegaMenu() {
                         </ul>
                       </div>
                     ))}
-                    <div className="flex flex-col items-center justify-center bg-neutral-100 rounded-md border-neutral-300 overflow-hidden grow">
+                    <div className="flex flex-col items-center justify-center overflow-hidden rounded-md bg-neutral-100 border-neutral-300 grow">
                       <img src={bannerDetails.image.src} alt={bannerDetails.title} className="object-contain" />
-                      <p className="mb-4 mt-4 px-4 text-center typography-text-base font-medium">
+                      <p className="px-4 mt-4 mb-4 font-medium text-center typography-text-base">
```

Comment:
Do you have an util like Prettier for Tailwind classes sorting enabled? Does it sort classes in a particular order? We may consider using it as the team to unify and prevent this kinda "changes" in the future.

Reply:
Yes, I used to have it. It was removed form this workspace but it seems it's enabled again. I'll check it and remove

Reply:
I have it as well but I guess its better to not have things like that

Reply:
@AdamPawlinski /\ :)

Reply:
I mean, imo it'd be nice to have it in the project, so it'd force all the classes to look exactly the same no matter of the author. 🤔